README.md

```diff
@@ -1 +1,15 @@
-TODO
+# Amazon ECS CNI Plugins
+
+[![Build Status](https://travis-ci.org/aws/amazon-ecs-cni-plugins.svg?branch=dev)](https://travis-ci.org/aws/amazon-ecs-cni-plugins)
```

Comment:
`dev` for now until we merge code into `master`, which I think we should do once this PR is merged.